vanilla-framework/vanilla-framework
# Update ruby gems

## Done

Github was complaining about a security vulnerability with the Jekyll gem so I ran an upgrade on all the gems.

<img width="749" alt="screenshot 2018-10-09 at 20 54 14" src="https://user-images.githubusercontent.com/505570/46694758-96b2c080-cc05-11e8-8c3b-de6aa1c5ec0f.png">
 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Verify everything works as expected
